modules/react/icon/lib/SystemIcon.tsx

```diff
@@ -159,10 +159,10 @@ export const systemIconStencil = createStencil({
       height: cssVar(height, cssVar(size, system.space.x6)),
     },
     '& .wd-icon-fill': {
-      fill: cssVar(color, base.licorice200),
+      fill: cssVar(color, system.color.icon.default),
     },
     '& .wd-icon-accent, & .wd-icon-accent2': {
-      fill: cssVar(accentColor, cssVar(color, base.licorice200)),
+      fill: cssVar(accentColor, cssVar(color, system.color.icon.default)),
```

Comment:
Change to fg?

Reply:
yeah lets do it, I was hoping to talk to design to see if they had an opinion on `fg` vs these tokens

Reply:
i guess we can move out from text and system for parts of our components